### ISSUE TITLE
OSError: [Errno 38] Function not implemented

### ISSUE THREAD

**Issue by oerp-odoo** [giampaolo/pyftpdlib#496]
I encountered an issue with `pyftpdlib` when it is being used on Ubuntu `18.04` (it works fine with Ubuntu `16.04`).

Doing this:

```python
from pyftpdlib.servers import FTPServer
```
Throws this:
```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/odoo/.local/lib/python3.6/site-packages/pyftpdlib/servers.py", line 520, in <module>
    class MultiprocessFTPServer(_SpawnerBase):
  File "/home/odoo/.local/lib/python3.6/site-packages/pyftpdlib/servers.py", line 524, in MultiprocessFTPServer
    _lock = multiprocessing.Lock()
  File "/usr/lib/python3.6/multiprocessing/context.py", line 67, in Lock
    return Lock(ctx=self.get_context())
  File "/usr/lib/python3.6/multiprocessing/synchronize.py", line 162, in __init__
    SemLock.__init__(self, SEMAPHORE, 1, 1, ctx=ctx)
  File "/usr/lib/python3.6/multiprocessing/synchronize.py", line 59, in __init__
    unlink_now)
OSError: [Errno 38] Function not implemented
```

Linux:

No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.2 LTS
Release:        18.04
Codename:       bionic

Python 3.6.7
pyftpdlib 1.5.4

**Comment by lurch** [comment]
Looks similar to https://github.com/tqdm/tqdm/issues/466 and https://bugzilla.redhat.com/show_bug.cgi?id=1511200 and https://bugs.python.org/issue26924 ?

**Comment by oerp-odoo** [comment]
Apparently it seems it is related with `multiprocessing` and `Lock`. Gives same error, doing:
```python
import multiprocessing
multiprocessing.Lock()
```
```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/multiprocessing/context.py", line 67, in Lock
    return Lock(ctx=self.get_context())
  File "/usr/lib/python3.6/multiprocessing/synchronize.py", line 162, in __init__
    SemLock.__init__(self, SEMAPHORE, 1, 1, ctx=ctx)
  File "/usr/lib/python3.6/multiprocessing/synchronize.py", line 59, in __init__
    unlink_now)
OSError: [Errno 38] Function not implemented
```

**Comment by giampaolo** [comment]
Wow! That's weird. I fixed it in  004b975. Please confirm it works and I'll make a new release tomorrow.

**Comment by oerp-odoo** [comment]
@giampaolo I tested on our CI environment (where issue occurred) and I can confirm it does not raise an exception anymore (and unittests that use `pyftpdlib` also pass)
